src/components/Filters/Filters.jsx

```diff
@@ -64,7 +60,7 @@ export default function Filters() {
           <Link
             key={category.id}
             className="FilterItem"
-            to={`/search?categories=${category.id}`}
+            to={`/search?categories=${category.id}`} // should this be `/category/${category.id}`? Or no link at all?
```

Comment:
doing this search usually just redirects back to our products page (I assume since no /search route has been defined)

Reply:
I was planning to pull the query parameters from the products page